core/utils/substrate.ts

```diff
@@ -0,0 +1,4 @@
+import { ScProvider } from "@polkadot/rpc-provider/substrate-connect/index.ts";
```

Comment:
All of the imports should come from `useink/core/mod.ts` (You can re-export them in `core/types/substrate-connect.ts` + `core/types/mod.ts`)

Reply:
done, thx